Tasks/AzureTestPlanV0/testPlanData.ts

```diff
@@ -27,13 +26,12 @@ export interface ManualTestRunData {
 export async function getTestPlanData(): Promise<TestPlanData> {
 
     const testPlanDataResponse: TestPlanData = {
-        automatedTestPointIds: [],
         listOfFQNOfTestCases: [],
-        manualTestPointIds: [],
         testPlanId: 0,
         testSuiteIds: [],
         testConfigurationId: 0,
-        listOfTestPointDetails: []
+        listOfManualTestPoints: [],
```

Comment:
nit: since we storing results objects lets rename these

Reply:
Will address all of the nit changes in next iteration